assignments/assignment-6/public/scripts.js

```diff
@@ -1,8 +1,21 @@
+var currentScoreAmount = 300;
+var team1Score = 0;
+var team2Score = 0;
+var team3Score = 0;
+var team4Score = 0;
+var pointsClicked = false;
+
 var DATA = [];
 
 function startGame() {
+  if (checkBoxesCheck() == true && teamNameCheck() == true) {
```

Comment:
I think we can just do:
```
if (checkBoxesCheck() && teamNameCheck()) {
```

Reply:
Yeah, that should work, I'll make the change.